trio/_tests/verify_types_darwin.json

```diff
@@ -76,7 +76,7 @@
     ],
     "otherSymbolCounts": {
       "withAmbiguousType": 0,
-      "withKnownType": 682,
+      "withKnownType": 680,
```

Comment:
if checking manually (I realize now I removed generating `full_output.json` feel free to add that back if it helps), what's the removed symbols?

Reply:
It'd be the `nursery.start()` protocol, and its `__call__` method.

Reply:
:+1: 
We might want to keep a version of these `json` files around in the end after all, though just as the list of symbols. Kinda similar to what `test_exports` is doing but not quite. I'm not sure if dropping private symbols is ever really a problem, but could be good to keep track of it.

Reply:
Not really, as long as everything works (= tests pass). If it's private, by definition it can change whenever we want.

Reply:
Well, main reason would be checking the public types - the private types mostly just incidental but I don't think it hurts (at least if the updating of the file can be automated in pre-commit or so)